frontend/public/components/_route.scss

```diff
@@ -4,6 +4,11 @@
 }
 
 .co-m-route-warnings {
+  background-color: #f7e2e2;
+  border: solid 2px $red;
+  color: $color-red-error;
+  font-weight: 700;
+  padding: 6px;
```

Comment:
Is an existing class with this style we can use?

@tlwu2013 Just confirming this is the error style you had recommended (see screenshots).

Reply:
There is something in 'Applications -> Cluster Service Versions -> Overview' but I couldn't find the exact code.

Reply:
> is the green "Ready" using PF green 400? I think it should be per #123.

hi, yes, I'm going to change to use pf green 400

>Is the error text intentionally bold when inside the red area?

yes, font-weight 700, probably shouldn't be.  I was trying to only bold the first line/topic sentence per @tlwu2013 mockup in JIRA.  The way it's implemented, some additional coding would be needed to isolate the first sentence.  

We might not inline the error here, looking into @spadgett's suggestion:
> We might want to use the Conditions component from #97 instead of inlining the error here. It has more detail and should have the same basic info about accepted / rejected routes.